components/CollectiveNavbarActionsMenu.js

```diff
@@ -238,6 +314,19 @@ const CollectiveNavbarActionsMenu = ({
           </StyledButton>
         </MenuPopover>
       </Box>
+      <AddFundsModal
+        collective={collective}
+        host={collective}
+        show={hasAddFundsModal}
+        setShow={showAddFundsModal}
+        onClose={() => showAddFundsModal(null)}
+      />
```

Comment:
If I remember correctly, you initially added these into the conditional blocks (`{callsToAction.addFundsToOrganization && (`). Since these two components are imported with `dynamic`, it was good because you only fetched the code when using them.

In the current version, the `dynamic` has no effect since we're always rendering the modals, regardless of your permissions. What was the motivation behind this change?